howso/return_typing.amlg

```diff
@@ -424,6 +424,10 @@
 									type "boolean"
 									description "Flag indicating if the label should be given the active session for the session parameter automatically from clients."
 								}
+								"protected" {
+									type "boolean"
+									description "Flag indicating this method should not be publicly available in the API."
```

Comment:
the other description refer to the "label" rather than the method, could you change this to use "label"?